versions.props

```diff
@@ -15,6 +15,8 @@ com.googlecode.java-diff-utils:diffutils = 1.3.0
 com.puppycrawl.tools:checkstyle = 10.3.2
 org.checkerframework:* = 3.23.0
```

Comment:
Should we bump this to 3.25.0 and remove the pin below (and maybe sort these)?

Reply:
Ah, the dependency upgrade excavator is stuck, good catch! I agree that we shouldn't pin a more specific version, keeping these in sync is helpful.